tests/test_helpers.py

```diff
@@ -515,6 +515,54 @@ def test_proxies_from_env_http_with_auth(mocker) -> None:
     assert proxy_auth.encoding == "latin1"
 
 
+# --------------------- get_env_proxy_for_url ------------------------------
```

Comment:
Could you also add a test for the positive outcome? You could use `@pytest.mark.parametrize` to run it with several sets of params if necessary.

Reply:
The positive case was the last one https://github.com/aio-libs/aiohttp/pull/4445/files#diff-08a6944653efc734f4c8cb856fc854b9f21d655ca69796af2b170c53bff7919eR556

I could add more than one